src/components/balance/modals/ModalTransferAmount.vue

```diff
@@ -88,7 +88,7 @@
                   v-model:selAddress="toAddress"
                   :all-accounts="allAccounts"
                   :all-account-names="allAccountNames"
-                  role="toAddress"
+                  :role="Role.ToAddress"
```

Comment:
![bandicam 2021-11-14 13-28-27-315](https://user-images.githubusercontent.com/40356749/141681169-e21219d0-3c5a-4689-a23e-19c45a16af8d.jpg)

The account drop-down option does not close when I paste an address. The input component should search if the user is inputting a known address or close the drop-down if there is no match